keyboards/keycapsss/kimiko/kimiko.c

```diff
@@ -75,4 +77,220 @@ led_config_t g_led_config = { {
     4, 4, 1, 1, 1, 1
 } };
 
-#endif // RGB_MATRIX_ENABLE
\ No newline at end of file
+#endif // RGB_MATRIX_ENABLE
+#endif // KEYBOARD_keycapsss_kimiko_rev1
+
+#ifdef KEYBOARD_keycapsss_kimiko_rev2
```

Comment:
This all should be moved to `rev2.c`, rather than be added here.

Reply:
Moved the code to rev2.c as you suggested.
Also migrate some stuff to info.json